test/parse-outputs/test_parse_outputs.py

```diff
@@ -37,3 +38,17 @@ def test_empty_file(self):
 
             runner.parse_output(algo, test_file, out_file)
             assert filecmp.cmp(OUTDIR + f"{algo}-empty-pathway.txt", EXPDIR + f"empty-pathway-expected.txt", shallow=False)
+
+    def test_oi2_miss_insolution(self):
+        test_file = OI2_EDGE_CASES_INDIR + f"omicsintegrator2-miss-insolution-raw-pathway.txt"
+        out_file = OUTDIR + f"omicsintegrator2-miss-insolution-pathway.txt"
+
+        runner.parse_output('omicsintegrator2', test_file, out_file)
+        assert filecmp.cmp(out_file, EXPDIR + f"empty-pathway-expected.txt", shallow=False)
```

Comment:
Do we need to commit the two new expected output files used in this test case?

For some reason, it looks like GitHub Actions isn't running these new test functions.

Reply:
The pathways from expected are files that already existed prior to these new tests. 

I can look over why their not running in GIthub Actions, but they're running locally.

Reply:
I was wrong about GitHub Actions. I checked the raw log at https://productionresultssa4.blob.core.windows.net/actions-results/5976fa12-4dd4-4ef0-89ce-d3656415ccc0/workflow-job-run-78bf5d3d-47df-5be6-a66b-bcf82a771e6e/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-09-12T18%3A18%3A53Z&sig=pKSCvzCmaJD%2BeAxZSu7pHjnZ8%2FiZ7m59GJJ9P7MhZ1o%3D&ske=2024-09-13T03%3A38%3A36Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2024-09-12T15%3A38%3A36Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2024-05-04&sp=r&spr=https&sr=b&st=2024-09-12T18%3A08%3A48Z&sv=2024-05-04 and searched that to find
```
2024-09-04T23:12:03.2930468Z test/parse-outputs/test_parse_outputs.py::TestParseOutputs::test_parse_outputs PASSED
2024-09-04T23:12:03.3004279Z test/parse-outputs/test_parse_outputs.py::TestParseOutputs::test_empty_file PASSED
2024-09-04T23:12:03.3027440Z test/parse-outputs/test_parse_outputs.py::TestParseOutputs::test_oi2_miss_insolution PASSED
2024-09-04T23:12:03.3053933Z test/parse-outputs/test_parse_outputs.py::TestParseOutputs::test_oi2_wrong_order PASSED
```